File: python/k번째 수.py
def solution(array, commands):
    answer = []
    
    for command in commands:
        # i, j, k 꺼내기
        k = command.pop()
        j = command.pop()
        i = command.pop()
        
        _array = [] # i ~ j번째 숫자를 담을 배열
        for index in range(i, j + 1):
            print(index)
            _array.append(array[index - 1])
        
            
        _array.sort() # 정렬
        answer.append(_array[k - 1]) # K번째 수 append

    return answer

File: java/k번째수.java
import java.util.*;

class Solution {
    private static List<Integer> sliceArr;
    
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int idx = 0;
        
        for(int[] command: commands){
            sliceArr = new ArrayList<>();
            for(int i=command[0]-1; i<command[1]; i++){
                sliceArr.add(array[i]);
            }
            
            Collections.sort(sliceArr);
            
            answer[idx++] = sliceArr.get(command[2] - 1);
        }

        return answer;
    }
}
